Leanplum-SDK/Classes/LPInternalState.h

```diff
@@ -9,8 +9,6 @@
 #import "LPRegisterDevice.h"
 #import "LPActionManager.h"
 
-#pragma mark - LPInternalState interface
```

Comment:
I don't mind removing these but note that these are here to make it look cleaner on quick function view.